backend/routes/auth.go

```diff
@@ -8,8 +8,9 @@ import (
 
 func AuthRoute() chi.Router {
 	r := chi.NewRouter()
-	r.Post("/register/request", controller.RegisterRequest)
-	r.Post("/register/response", controller.RegisterResponse)
-	r.Get("/keys", controller.GetKeys)
+	r.Post("/register/request/{userId}", controller.RegisterRequest)
```

Comment:
I added the user to the route because a) we don't really care if we expose this data and b) the webauthn functions need to read from the POST request body, and because the request body is an IO stream, it's consumed whenever it is read. Ie: it is annoying to read the request body multiple times (once for getting user id, once for getting remaining webauthn data)